cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -107,6 +109,86 @@ def test_errored_course_global_staff(self):
             courses_list_by_groups, __ = _accessible_courses_list_from_groups(self.request)
             self.assertEqual(courses_list_by_groups, [])
 
+    @ddt.data(
+        (ModuleStoreEnum.Type.split, 5),
+        (ModuleStoreEnum.Type.mongo, 3)
+    )
+    @ddt.unpack
+    def test_staff_course_listing_performance(self, default_store, mongo_calls):
+        """
+        Create courses and compare the time to fetch accessible courses for the staff user and
```

Comment:
Update doc string. All you need to test in this case is that it takes some number of mongo calls to call _staff_accessible_course_list(self.request)